Packages/com.unity.inputsystem/package.json

```diff
@@ -18,8 +18,5 @@
 		"touch",
 		"vr",
 		"xr"
-	],
-	"dependencies": {
-		"com.unity.ugui": "1.0.0"
-	}
-}
```

Comment:
With this gone, do we need to add the ugui dependency to Packages/manifest.json? We do want uGUI to be pulled into the InputSystem project as a whole.

Reply:
> 
> 
> With this gone, do we need to add the ugui dependency to Packages/manifest.json? We do want uGUI to be pulled into the InputSystem project as a whole.

UGUI did not become a package until 2019.2 so we can't really add it as the project is 2019.1 at the moment. It will be automaticvally added when the project upgrades. If it doesn t then the project will fail as it has a lot of ugui code in it.

Reply:
Ah right, forgot about the automatic upgrade for the 2019.1 project. All good then :)